pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -177,8 +179,8 @@ public QueryExecutorImpl(PGStream pgStream,
   }
 
   @Override
-  public int getProtocolVersion() {
-    return 3;
+  public ProtocolVersion getProtocolVersion() {
```

Comment:
I wonder if it breaks previous api

Reply:
I don't think so, the API is pretty much internal. That said we will see.